tests/test_generic_plotter.py

```diff
@@ -43,6 +48,16 @@ def test_plotter_add_pd():
     pl.plot(sphere)
     pl.show()
 
+@pytest.mark.skipif(IN_GITHUB_ACTIONS, reason="Qt breaks CICD.")
+def test_plotter_pyvistaqt():
+    """Adds polydata to the plotter."""
+    qt_backend = PyVistaBackend(use_qt=True)
+    pl = Plotter(backend=qt_backend)
+    sphere = pv.Sphere()
+    pl.plot(sphere)
+    # PyVista QT show() breaks PyTest, so we avoid it.
+    qt_backend.close()
+
```

Comment:
Ah you managed to test it. Then we might not need to remove the system deps you were installing, my bad